.ci/release.sh

```diff
@@ -38,3 +38,6 @@ fi
 
 echo "--- Release the binaries to Maven Central :maven: [./mvnw ${GOAL})] ${DRY_RUN_MSG}"
 ./mvnw -V -s .ci/settings.xml -Pgpg clean $GOAL -DskipTests --batch-mode | tee release.txt
+
+echo "--- Archive the target folder with jar files"
+find . -type d -name target -exec find {} -name '*.jar' -print0 \; | xargs -0 tar -cvf "${TARBALL_FILE:-dist.tar}"
```

Comment:
Use `.ci/published-artifacts-list.sh` script added in https://github.com/elastic/ecs-logging-java/pull/251.